schema/fields.go

```diff
@@ -276,16 +276,40 @@ var SupportedFieldProperties = container.NewHashSet(
 
 // Indexes is to wrap different index that a collection can have.
 type Indexes struct {
-	PrimaryKey     *Index
-	SecondaryIndex *Index
+	All []*Index
```

Comment:
So All returns all the secondary indexes, right?

Reply:
Yes. correct